### ISSUE TITLE
Cannot tab into any of the elements within the file previewer 

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#1535]
When using a screen reader can not tab into any of the elements/buttons within the previewer e.g. close button, maximise, minimise etc. 

![screen shot 2017-01-24 at 15 14 33](https://cloud.githubusercontent.com/assets/13200338/22253053/227e8dc8-e248-11e6-9422-259deda93be6.png)
